frameworks/src/foundry/mod.rs

```diff
@@ -145,9 +145,9 @@ impl ParseLow for Foundry {
         "assert*",
         "vm.expectEmit",
         "vm.expectRevert",
-        "vm.prank",
-        "vm.startPrank",
-        "vm.stopPrank",
+        "vm.expectCall",
```

Comment:
```suggestion
        "vm.expect*",
```
And remove the `vm.expectEmit` and `vm.expectRevert` lines.

I think this is a bug introduced by refactoring. As you can see, this doesn't match what is in the documentation: https://github.com/trailofbits/necessist/blob/45c1abba36889159badc88ec0dc5c6c57fa15868/README.md?plain=1#L162